WordPressAuthenticator/Sources/Signin/LoginLinkRequestViewController.swift

```diff
@@ -36,7 +37,9 @@ class LoginLinkRequestViewController: LoginViewController {
 
         let email = loginFields.username
         if email.isValidEmail() {
-            gravatarView?.downloadGravatarWithEmail(email, rating: .x)
+            Task {
+                try await gravatarView?.setGravatarImage(with: email, placeholder: .gravatarPlaceholderImage, rating: .x)
```

Comment:
I wasn't able to make this code run even though I followed the email link flow. Maybe I am missing something.

But this is the same method we call from `GravatarEmailTableViewCell.swift`. And I was able to test that.